examples/chrome-example-test.js

```diff
@@ -22,7 +22,11 @@ async function main() {
     const text = msg.text();
     console.log(`>> ${text}`);
     if (msg.type() === 'error') {
-      errored = true;
+      // On initial load, the workspace 404 is expected.
```

Comment:
This is not a very satisfying solution, so I'm not sure whether we should handle it this way.

Another option we have is actually writing a file called `fooexample-94b7.jupyterlab-workspace` which contains `{"data":{},"metadata":{"id":"/foo/example"}}` into `examples/app/build/workspaces` when `main.py` runs so that it never 404s.

Reply:
I'm happy to have this be the place we identify known issues rather than add hacks to the examples.

Reply:
If we change the behavior of `jupyterlab_server` as per this PR, we can avoid this special-case behavior.

https://github.com/jupyterlab/jupyterlab_server/pull/64